index.html

```diff
@@ -206,41 +249,43 @@ <h3 class="card-title">Skills</h3>
     </section>
 
   </Main>
+
   <footer class="contact-form">
-    <div class="footer-text">
-      <h2 class="title">Contact Me</h2>
+    <h2 class="title">Contact Me</h2>
+    <div class="footer-content">
```

Comment:
- [ ] In the "I can help you build..." text, please kindly correct the font style by checking the template.
- [ ] In the form placeholders, you can change the font style and family by using the pseudoclass `::placeholder`.  More info [here](https://www.w3schools.com/cssref/sel_placeholder.asp).
- [ ] In the Figma template, the border for the `textarea` input is a different color (green).  Let's kindly correct it.
- [ ] There is a horizontal line that should be over the icons, but it's missing 😢.  Please kindly add it.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/149324285-83319166-b83d-4cb9-bb95-bffe50e16314.png) | ![image](https://user-images.githubusercontent.com/57421823/149324344-a272d326-07c3-47ab-bd5b-9f26c4bdbc85.png)

That way, this particularly beautiful project will be finished!! 💯

Reply:
Thanks,
This is a nicely detailed review.
I'll make sure to make all the changes
:smile: